core-ui/src/components/Predefined/Create/Gateways/Forms/HostsForm.js

```diff
@@ -14,6 +14,7 @@ export const HostsForm = ({ server = {}, servers, setServers }) => {
     <TextArrayInput
       advanced
       required
+      ariaLabel="Host"
```

Comment:
Shouldn't Aria label be taken from i18n as well? this is formally meant for accessibility purposes, after all

Reply:
(this applies to all cases of ariaLabel appearing in the PR)